.github/workflows/test-suite.yml

```diff
@@ -54,6 +54,20 @@ jobs:
          done
          echo "skip_ci=$SKIP_CI" >> $GITHUB_OUTPUT
 
+  lockbud:
+    name: lockbud
+    runs-on: ubuntu-latest
+    container:
+      image: eserilev/lockbud:latest
```

Comment:
Is this a blocker to merging?

Reply:
Updated to point to `sigmaprime/lockbud`